src/library/Menu/MenuItem.js

```diff
@@ -73,12 +90,12 @@ export default class MenuItem extends PureComponent<MenuItemProps> {
   }
 
   getItemProps: MenuItemPropGetter = (props = {}) => {
-    const { disabled, onClick, render, ...restProps } = props;
+    const { disabled, onClick, render, tabIndex, ...restProps } = props;
 
     return {
       ...(render ? restProps : {}),
       disabled,
-      tabIndex: disabled ? '-1' : 0,
+      tabIndex: tabIndex || (disabled ? '-1' : 0),
```

Comment:
I think this needs a bit more adjustment. It is causing a tabIndex of 0 to be added to Select/Dropdown items and breaking tab navigation.

Reply:
Thanks for the catch. Changed it to `tabIndex !== undefined ? tabIndex : disabled ? '-1' : 0`

Reply:
Perhaps we can add a couple quick regression tests to Select and Dropdown? E.g. render open, keydown tab, assert active element